deploy/helm/charts/values.yaml

```diff
@@ -93,6 +93,8 @@ nfsStorageClass:
   nfsServerType: kernel
   isDefaultClass: false
   backendStorageClass: ""
+  mountOptions:
+    - vers=4.1
```

Comment:
Keeping a provision for using mountOptions is ok, but I'd say enforcing a particular default like vers 4.1 won't be desired. A user deploying nfs server for a performant workload may prefer vers 3 for example, or some clients by default start negotiating highest supported vers i.e. 4.2.

Reply:
@pentago Please revert the changes done in values.yaml based on above comment.

Reply:
@dsharma-dc This doesn't seem to move forward. Would you be open to a separate PR for this? I've released a patch version on my fork and it works perfectly. https://github.com/openebs/dynamic-nfs-provisioner/commit/18e41e03356442b6306b558659ec2917c5d3a1f7

Reply:
@beschoenen I have created #164 to apply this change.

Reply:
Great, thanks!